transport/mqtt/driver.go

```diff
@@ -103,16 +114,26 @@ func New(config arc_config.Config, isServer bool) (*MQTTClient, error) {
 		connected:     false,
 		isServer:      isServer,
 		subscriptions: make(map[string]subscription),
+		lastSeenError: nil,
+		// allow 5 reconnects every 10 sec
```

Comment:
That seems very permissive, I would go with something like 1 reconnect per 10 seconds or something

Reply:
done!